src/gui/editors/AutomationEditor.cpp

```diff
@@ -862,6 +925,56 @@ void AutomationEditor::mouseMoveEvent(QMouseEvent * mouseEvent )
 				}
 			break;
 			}
+			case EditMode::EditTangents:
+			{
+				// If we moved the mouse past the beginning correct the position in ticks
+				posTicks = qMax(posTicks, 0);
+
+				if (m_mouseDownLeft)
+				{
+					if (m_action == Action::MoveTangent)
+					{
+						timeMap & tm = m_clip->getTimeMap();
+						timeMap::iterator it = tm.find(m_draggedTangentTick);
```

Comment:
```suggestion
						auto it = tm.find(m_draggedTangentTick);
```
I'd modernize this with `auto`, though this is just me nitpicking